go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/coreos/pkg
```

Comment:
I think we should technically start at `v5` here, because we already tagged version 4.

At the same time, I fear that is going to trigger a chain reaction of bad things because Go handles modules versions greater than 1 in a funky way (i.e. will need to adjust their imports).

Reply:
Ah, yeah.  Since this repo is effectively in deep maintenance mode, should we try to avoid introducing `go.mod` here?

Reply:
If the only winning move is not to play, then yes I think we can stick to that strategy (and hope that eventually all consumers dry out).

Reply:
Okay, I've removed `go.mod` from the root of the repo and rolled back all dependency changes.  Instead, I've put a private version in `.ci/go.mod` and taught the build/test scripts to use it.  That should allow us to test with modern infrastructure without publicly exposing a module.